cmd/juju/commands/resolved_test.go

```diff
@@ -87,22 +87,31 @@ var resolvedTests = []struct {
 	}, {
 		args: []string{"multi-series/4", "roflcopter"},
 		err:  `unrecognized args: \["roflcopter"\]`,
+	}, {
+		args: []string{"multi-series/5", "--all"},
```

Comment:
This code should signal that there's something missing in the implementation - it doesn't make any sense to all passing a single unit name *and* --all. It's one or the other. That aside, I'm not sure what this extra test as written was intended to achieve.